### ISSUE TITLE
loss in  'eval or test 'mode  ? version in spconv 2.1.7, pytorch 1.9

### ISSUE THREAD

**Issue by mengshiyu0109** [traveller59/spconv#378]
Dear author

when the model is training, the loss value is normal , the numerical value almost starts from about 1 . And the model can run  well.
But when it is in the eval mode, my loss value got nearly about to 200.  and  in some batch,  it is  '[ ]'.It is really large or to zero. 

I 'm very confused, how can I do ? 

thanks for everything.

**Comment by mengshiyu0109** [comment]
BTW, about loss is  [ ], it has never happened in version 2.1.6  ,under training mode.

But in version 2.1.7, I got the  '[ ]' loss  or large loss in eval. 
the code is same.

**Comment by FindDefinition** [comment]
I need more information to debug. 
1. you can set ```algo``` in all conv/algo to ```spconv.ConvAlgo.Native``` and check whether your problem is resolved.
2. your network architecture, approximately

**Comment by mengshiyu0109** [comment]
Dear author,

Thanks for your prompt reply. 
My model is just 4 layers of downsampling and 4 layers of upsampling, similar to the 3D structure of unet.
I thought of a possible cause, is it due to overfitting?  I will try this first.

If it is not work, I will try the "algo in all conv/pool to spconv.ConvAlgo.Native" way .

thanks!

**Comment by FindDefinition** [comment]
I can't reproduce this problem in waymo point cloud dataset (UNet-like segmentation), the only one difference between train and eval is ```get_indice_pairs_implicit_gemm``` function, you can set them to ```True``` by modify code in pip-installed spconv path.
If the ```algo``` argument in conv/pool is set to ```spconv.ConvAlgo.Native```, there is no difference between train and eval.
